templates/index.html

```diff
@@ -3,18 +3,39 @@
 {% block title%}Home Page{% endblock%}
 {% block content %}
 
-<div class="container">
+
+<div class="container" id="poop">
   <div class="row">
     <div class="col-md-10 col-lg-8 mx-lg-auto mx-md-auto">
 
       <h1 class="pt-5 pb-2">Welcome to CPM Detail</h1>
 
-      {{ render_form(form) }}
+      {{ render_form(form, id="poppy") }}
```

Comment:
Ooops. When I get frustrated I start adding random while I work on a solution. Since the getelementID call requires an ID  and the form didn't have one I found the solution. Should probs be NameForm. Also I think I called something else poop now that I think of it. XD.

Not something I was focused on last night, but maybe some Arch convo around what was added to the color.css file to make this work as it is technically not a color. Should there be other Static X.CSS by function?

Reply:
Yea, I think I breezed past that the last time we talked, but yes there should be separate files for that kind of stuff. Good call